### PR TITLE
Move search button in top-bar next to tabs

### PR DESCRIPTION
**Pull request by Tristan-WorkGH**
Also move code from commons-ui to GridStudy

Don't need gridsuite/commons-ui#325 to be tested.

### CODE REVIEW

File: src/components/app-top-bar.js [hunk 1]
```diff
@@ -382,8 +400,37 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
...
+                                <IconButton
+                                    color="inherit"
+                                    aria-label="search"
+                                    size="large"
+                                    onClick={() => setDialogSearchOpen(true)}
+                                >
+                                    <Search />
+                                </IconButton>
+                            </Tooltip>
+                        </Box>
```
Comment (lilian-houdelet) [inline src/components/app-top-bar.js:433]: Il y a une différence entre les deux boutons search ?

**Review by lilian-houdelet — CHANGES_REQUESTED** [review]

File: src/components/app-top-bar.js (hunk 1 above)
Comment (Tristan-WorkGH) [reply src/components/app-top-bar.js:433]: oups, I forget to remove it!

Yes they don't have the same apparence (mainly one is ~sqaure~ retangle and the other is round.

File: src/components/app-top-bar.js [hunk 2]
```diff
@@ -382,8 +400,21 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
                                 return <Tab key={tabName} label={label} />;
                             })}
                         </Tabs>
-                    )}
-                    {studyUuid && (
+                        <Box sx={styles.searchButton}>
+                            <Tooltip
```
Comment (dbraquart) [inline src/components/app-top-bar.js:404]: This tooltip was not here before.
Maybe you should ask a PO to confirm ?

**Review by dbraquart — COMMENTED** [review]
code: ok
tests: works fine

just a question about the added tooltip

File: src/components/app-top-bar.js (hunk 2 above)
Comment (Tristan-WorkGH) [reply src/components/app-top-bar.js:404]: I confirmed with a PO, it's OK.

**Review by lilian-houdelet — APPROVED** [review]
Code OK
Test OK

**Review by dbraquart — APPROVED** [review]

**Pull request merged**